cl_tf2/src/cl-transforms-plugin/package.lisp

```diff
@@ -36,4 +36,4 @@
    ;; point-stamped
    point-stamped make-point-stamped copy-point-stamped point
    ;; pose-stamped
-   pose-stamped make-pose-stamped copy-pose-stamped pose))
```

Comment:
Fixed in https://github.com/fairlight1337/roslisp_common/commit/f81277eda159271cb3fe4c93f7ced4632a3d3d08.